tribe-ext-main-blog-loop/index.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Main Blog Loop
+ * Description:     Additional options for including events in the main blog loop.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Main_Blog_Loop
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+// Do not load unless Tribe Common is fully loaded.
```

Comment:
Any reason no space between these checks?

Instead of this:
```
}
// Do not load unless Tribe Common is fully loaded.
```
I'd recommend adding a line there to maintain the consistency of vertical spacing, like this:
```
}

// Do not load unless Tribe Common is fully loaded.
```